src/main/java/Duke.java

```diff
@@ -1,10 +1,81 @@
+import Event.Event;
+import Event.Deadline;
+import Event.ToDo;
+import Event.Task;
+import RuntimeSupport.DukeException;
+import java.util.Scanner;
+
 public class Duke {
+    static String BREAK_LINE = "____________________________________________________________";
+
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+
+        System.out.println(BREAK_LINE + "\nHello! I'm 550W.\nWhat can I do for you?\n" + BREAK_LINE);
+        Task[] tasks = new Task[100];
+        int count = 0;
+        Scanner in = new Scanner(System.in);
+        String line = "";
+
+        while (true) {
```

Comment:
You could consider refactoring this method to separate methods to adhere to this code quality [guideline](https://nus-cs2113-ay2324s2.github.io/website/se-book-adapted/chapters/codeQuality.html#avoid-long-methods)